src/releaseAndTag.js

```diff
@@ -107,7 +106,7 @@ const createTempDirectory = async (name) => {
   if (exists) {
     throw new Error(`Tried to create directory, but it already exists: ${dir}`);
   }
-  mkdirp(dir);
```

Comment:
Hmmm...based on the way the rest of this is written, I assumed this was synchronous and replaced it with `fs.mkdirSync()` but this is async. I wonder if this is a second potential race condition.